Pipfile

```diff
@@ -31,6 +31,10 @@ pytest-mypy = "*"
 typeshed = "*"
 podman-compose = "*"
 sqlalchemy-stubs = "*"
+black = "*"
```

Comment:
Shouldn't we update `https://github.com/thoth-station/storages/blob/master/requirements-test.txt`, but also don't we want to avoid pre-releases for production? @fridex

Reply:
+1